src/include/verbose.h

```diff
@@ -6,8 +6,20 @@
  * See the file LICENSE for redistribution information.
  */
 
+#include <stdint.h>
+
 #pragma once
 
+/**
+ * Structure to bundle verbose message identification details.
+ * This makes passing verbose context more organized.
+ */
+struct __wt_verbose_message_info {
+    uint32_t verb_log_id;              /* A unique ID for the specific log message. */
+    WT_VERBOSE_CATEGORY verb_category; /* The verbose category (e.g., WT_VERB_API). */
+    WT_VERBOSE_LEVEL verb_level;       /* The verbosity level (e.g., WT_VERBOSE_INFO). */
```

Comment:
What do you think of the following?
```suggestion
    uint32_t id;              /* A unique ID for the specific log message. */
    WT_VERBOSE_CATEGORY category; /* The verbose category (e.g., WT_VERB_API). */
    WT_VERBOSE_LEVEL level;       /* The verbosity level (e.g., WT_VERBOSE_INFO). */
```

Reply:
Fixed in 442731e482279be6cd5e27d6925b013cb772843f